src/main/java/com/jamieswhiteshirt/reachentityattributes/mixin/ServerPlayNetworkHandlerMixin.java

```diff
@@ -19,11 +22,20 @@ abstract class ServerPlayNetworkHandlerMixin implements ServerPlayPacketListener
      *
      * <p>Attack range is further checked in {@link PlayerEntityInteractionHandlerMixin}.
      */
-    @ModifyConstant(
-        method = "onPlayerInteractEntity(Lnet/minecraft/network/packet/c2s/play/PlayerInteractEntityC2SPacket;)V",
-        require = 2, allow = 2, constant = @Constant(doubleValue = 36.0))
-    private double getActualAttackRange(final double attackRange, final PlayerInteractEntityC2SPacket packet) {
-        return ReachEntityAttributes.getSquaredReachDistance(this.player, attackRange);
+    @Redirect(
+        method = "onPlayerInteractEntity",
+        at = @At(value = "FIELD", target = "Lnet/minecraft/server/network/ServerPlayNetworkHandler;MAX_BREAK_SQUARED_DISTANCE:D", opcode = Opcodes.GETSTATIC)
+    )
+    private double getActualAttackRange() {
+        return ReachEntityAttributes.getSquaredReachDistance(this.player, 36.0);
+    }
+
+    @Redirect(
+        method = "onPlayerInteractBlock",
+        at = @At(value = "FIELD", target = "Lnet/minecraft/server/network/ServerPlayNetworkHandler;MAX_BREAK_SQUARED_DISTANCE:D", opcode = Opcodes.GETSTATIC)
+    )
+    private double getActualReachDistance() {
+        return ReachEntityAttributes.getSquaredReachDistance(this.player, 36.0);//ReachEntityAttributes.getSquaredReachDistance(this.player, reachDistance);
```

Comment:
I believe these should reference the actual MAX_BREAK_SQUARED_DISTANCE constant, in case some other mod (or even a future MC update) changes it.

Reply:
Oh, that's a good point! I'll add this in right away.